tests/unit/test-cmd/test.lint.js

```diff
@@ -3,157 +3,39 @@ import {it, describe} from 'mocha';
-      lint: (params = {}, options = {}) => {
-        return defaultLintCommand({
-          sourceDir: '/fake/source/dir',
-          ...params,
-        }, {
-          createLinter,
-          createFileFilter,
-          ...options,
-        });
-      },
-    };
-  }
-
-  it('creates and runs a linter', () => {
-    const {lint, createLinter, runLinter, lintResult} = setUp();
-    return lint().then((actualLintResult) => {
-      assert.equal(actualLintResult, lintResult);
-      assert.equal(createLinter.called, true);
-      assert.equal(runLinter.called, true);
-    });
-  });
-
-  it('fails when the linter fails', () => {
-    const createLinter = () => {
-      return {
-        run: () => Promise.reject(new Error('some error from the linter')),
-      };
-    };
-    const {lint} = setUp({createLinter});
-    return lint().then(makeSureItFails(), (error) => {
-      assert.match(error.message, /error from the linter/);
-    });
-  });
-
-  it('runs as a binary', () => {
-    const {lint, createLinter} = setUp();
-    return lint().then(() => {
-      const args = createLinter.firstCall.args[0];
-      assert.equal(args.runAsBinary, true);
-    });
-  });
-
-  it('sets runAsBinary according shouldExitProgram option', () => {
-    const {lint, createLinter} = setUp();
-    return lint({}, {shouldExitProgram: false}).then(() => {
-      const args = createLinter.firstCall.args[0];
-      assert.strictEqual(args.runAsBinary, false);
-    });
-  });
-
-  it('passes sourceDir to the linter', () => {
-    const {lint, createLinter} = setUp();
-    return lint({sourceDir: '/some/path'}).then(() => {
-      const config = createLinter.firstCall.args[0].config;
-      assert.equal(config._[0], '/some/path');
-    });
-  });
 
-  it('passes warningsAsErrors to the linter', () => {
-    const {lint, createLinter} = setUp();
-    return lint({warningsAsErrors: true}).then(() => {
-      const config = createLinter.firstCall.args[0].config;
-      assert.equal(config.warningsAsErrors, true);
-    });
-  });
-
-  it('passes warningsAsErrors undefined to the linter', () => {
-    const {lint, createLinter} = setUp();
-    return lint().then(() => {
-      const config = createLinter.firstCall.args[0].config;
-      assert.equal(config.warningsAsErrors, undefined);
-    });
-  });
-
-  it('configures the linter when verbose', () => {
-    const {lint, createLinter} = setUp();
-    return lint({verbose: true}).then(() => {
-      const config = createLinter.firstCall.args[0].config;
-      assert.equal(config.logLevel, 'debug');
-      assert.equal(config.stack, true);
-    });
-  });
-
-  it('configures the linter when not verbose', () => {
-    const {lint, createLinter} = setUp();
-    return lint({verbose: false}).then(() => {
-      const config = createLinter.firstCall.args[0].config;
-      assert.equal(config.logLevel, 'fatal');
-      assert.equal(config.stack, false);
-    });
-  });
-
-  it('passes through linter configuration', () => {
```

Comment:
I cannot find this test case anymore, it should probably be one of the tests of the new linter module.